cancer_ai/validator/competition_manager.py

```diff
@@ -84,17 +86,22 @@ def __init__(
         self.validator_hotkey = validator_hotkey
         self.db_controller = db_controller
         self.test_mode = test_mode
+        self.local_fs_mode = local_fs_mode
 
     def __repr__(self) -> str:
         return f"CompetitionManager<{self.competition_id}>"
 
     def log_results_to_wandb(
-        self, miner_hotkey: str, validator_hotkey: str, evaluation_result: ModelEvaluationResult
+        self,
+        miner_hotkey: str,
+        validator_hotkey: str,
+        evaluation_result: ModelEvaluationResult,
     ) -> None:
         winning_model_link = self.db_controller.get_latest_model(
             hotkey=miner_hotkey, cutoff_time=self.config.models_query_cutoff
         ).hf_link
         wandb.init(project=self.competition_id, group="model_evaluation")
+        # TODO make a data structure
```

Comment:
would wandb.log work with python structures?

Reply:
I did it with latest commit, it works. structures in models.py